# goal
int32 id
geometry_msgs/Pose goal_pose
geometry_msgs/Point[] target_points
DetectedObj[] detectedObj
uint32 current_waypoint

---
# result
bool success
uint32 new_current_waypoint
uint32 selected_target_index

---
# feedback